elide-core/src/main/java/com/yahoo/elide/core/RequestScope.java

```diff
@@ -166,6 +167,52 @@ public RequestScope(String path,
         if (transaction instanceof RequestScopedTransaction) {
             ((RequestScopedTransaction) transaction).setRequestScope(this);
         }
+
+        updateStatusCode = HttpStatus.SC_NO_CONTENT;
+    }
+
+    /**
+     * Create a new RequestScope with specified update status code
+     * @param path the URL path
+     * @param jsonApiDocument the document for this request
+     * @param transaction the transaction for this request
+     * @param user the user making this request
+     * @param dictionary the entity dictionary
+     * @param mapper converts JsonApiDocuments to raw JSON
+     * @param auditLogger logger for this request
+     * @param queryParams the query parameters
+     * @param securityMode the current security mode
+     * @param permissionExecutorGenerator the user-provided function that will generate a permissionExecutor
+     * @param updateStatusCode the response status code on successful path request
+     */
+    public RequestScope(String path,
```

Comment:
This should be the primary constructor and the other constructor should be `deprecated`.